src/containers/Storage/UsageFilter/UsageFilter.tsx

```diff
@@ -1,100 +0,0 @@
-import React from 'react';
-
-import type {SelectOption} from '@gravity-ui/uikit';
-import {Select} from '@gravity-ui/uikit';
-
-import {EntityStatus} from '../../../components/EntityStatus/EntityStatus';
-import {cn} from '../../../utils/cn';
-import {getUsageSeverityForEntityStatus} from '../utils';
-
-import i18n from './i18n';
-
-import './UsageFilter.scss';
-
-export interface UsageFilterItem {
-    threshold: number;
-    count: number;
-}
-
-interface UsageFilterProps {
-    className?: string;
-    value?: string[];
-    groups?: UsageFilterItem[];
-    onChange?: (value: string[]) => void;
-    debounce?: number;
-}
-
-const b = cn('usage-filter');
-
-export const UsageFilter = (props: UsageFilterProps) => {
```

Comment:
`UsageFilter` has been disabled in our UI for months. The problem it solved now is solved by grouping, it can be deleted